### PR TITLE
improve section on updating forked repository

### PR DESCRIPTION
**Pull request by svsven**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,33 +166,35 @@ now, when you add the files it ignores node_modules directory in your project.
...
+  4. Rebase your local branch, or merge the changes
+
+  **Rebase**
+  ```
+  git rebase upstream/master
+  ```
+
+  **Merge**
+  ```
+  git merge upstream/master
```
Comment (L04DB4L4NC3R) [inline README.md:197]: No need to merge after rebase

Comment (svsven) [reply README.md:197]: True, however the step states to either rebase or merge - but maybe that can be written a bit more clearer.

Comment (L04DB4L4NC3R) [reply README.md:197]: Agreed

Comment (svsven) [reply README.md:197]: Moved some text around and rephrased it a bit - should be more clear now.

**Pull request merged**